tests/frontmatter.test.js

```diff
@@ -0,0 +1,28 @@
+const glob = require('glob');
+const matter = require('gray-matter');
+
+describe('frontmatter tests', () => {
+  for (const mdFile of glob.sync('src/content/**/*.md')) {
```

Comment:
Did you try to use jest `.each` instead?

Reply:
I did look at it, but it's not a case of re-using the same data each time, the expected vs input is different. I think overall this approach is more direct and easier to understand.

Reply:
ok yep, works for me in any case 👍